Deprecation warnings on FFmpeg 4.4

The function `av_init_packet` was deprecated and now prints warnings during compilation 
```
/home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/parser.c:68:3: warning: ‘av_init_packet’ is deprecated [-Wdeprecated-declarations]
   68 |   av_init_packet(pkt);
      |   ^~~~~~~~~~~~~~
In file included from /usr/include/libavcodec/bsf.h:30,
                 from /usr/include/libavcodec/avcodec.h:44,
                 from /home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/parser.h:4,
                 from /home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/parser.c:1:
/usr/include/libavcodec/packet.h:488:6: note: declared here
  488 | void av_init_packet(AVPacket *pkt);
      |      ^~~~~~~~~~~~~~
/home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/decoder.c: In function ‘decode_with_dts’:
/home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/decoder.c:105:3: warning: ‘av_init_packet’ is deprecated [-Wdeprecated-declarations]
  105 |   av_init_packet(pkt);
      |   ^~~~~~~~~~~~~~
In file included from /usr/include/libavcodec/bsf.h:30,
                 from /usr/include/libavcodec/avcodec.h:44,
                 from /home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/decoder.h:4,
                 from /home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/decoder.c:1:
/usr/include/libavcodec/packet.h:488:6: note: declared here
  488 | void av_init_packet(AVPacket *pkt);
      |      ^~~~~~~~~~~~~~
/home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/decoder.c: In function ‘flush’:
/home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/decoder.c:147:3: warning: ‘av_init_packet’ is deprecated [-Wdeprecated-declarations]
  147 |   av_init_packet(pkt);
      |   ^~~~~~~~~~~~~~
In file included from /usr/include/libavcodec/bsf.h:30,
                 from /usr/include/libavcodec/avcodec.h:44,
                 from /home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/decoder.h:4,
                 from /home/bblaszkow/projects/membrane/plugins/video_merger/deps/membrane_h264_ffmpeg_plugin/c_src/membrane_h264_ffmpeg_plugin/decoder.c:1:
/usr/include/libavcodec/packet.h:488:6: note: declared here
  488 | void av_init_packet(AVPacket *pkt);
      |      ^~~~~~~~~~~~~~
```
See: https://patchwork.ffmpeg.org/project/ffmpeg/patch/20210305163339.63164-2-jamrial@gmail.com/

--- a/c_src/membrane_h264_ffmpeg_plugin/decoder.c
+++ b/c_src/membrane_h264_ffmpeg_plugin/decoder.c
@@ -98,7 +98,6 @@ UNIFEX_TERM decode(UnifexEnv *env, UnifexPayload *payload, int64_t dts, State *s
   UnifexPayload **out_frames = NULL;
   int64_t *best_effort_timestamps = NULL;
   pkt = av_packet_alloc();
-  av_init_packet(pkt);
   pkt->data = payload->data;
   pkt->size = payload->size;
   pkt->dts = dts;

--- a/c_src/membrane_h264_ffmpeg_plugin/parser.c
+++ b/c_src/membrane_h264_ffmpeg_plugin/parser.c
@@ -72,7 +72,6 @@ UNIFEX_TERM parse(UnifexEnv *env, UnifexPayload *payload, State *state) {
   memset(payload->data + old_size, 0, AV_INPUT_BUFFER_PADDING_SIZE);
 
   pkt = av_packet_alloc();
-  av_init_packet(pkt);
 
   uint8_t *data_ptr = payload->data;
   size_t data_left = old_size;

